content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+  the wall.
+
+  This field **only** appears on the event form if the *Social Marketing* app is installed, and an
+  *X* account has been added as a stream on the application.
+- :guilabel:`Template`: choose a pre-configured event template from the resulting drop-down menu.
+
+   Or, create a new one directly from this field, by typing in the name of the new template, and
+   selecting either: :guilabel:`Create` (which creates the template, and can be edited later) or
+   :guilabel:`Create and edit...` (which creates the template, and reveals a separate template page
+   to configure the template in greater detail).
+- :guilabel:`Tags`: add any corresponding tags to briefly describe the event (e.g. `Online`,
+  `Conference`, etc.). Multiple tags can be added per event.
+- :guilabel:`Organizer`: designate the organizer of the event.
+- :guilabel:`Responsible`: designate a user in the database to be responsible for this event.
+- :guilabel:`Company`: designate which company in the database to which this event is related. This
+  field **only** appears if working in a multi-company environment. If left blank, the event will
+  relate to all companies in the database.
+- :guilabel:`Website`: choose to restrict the publishing of this event to a specific website. If
+  this field is left blank, the event can be published on *all* websites in the database.
+- :guilabel:`Venue`: enter the pertinent event venue details in this field.
+- :guilabel:`Exhibition Map`: if desired, click the :guilabel:`Upload your file` button to Upload
+  a map of the event venue.
+- :guilabel:`Limit Registrations`: if this checkbox is ticked, a limit to the amount of
+  registrations is added to the event, and that desired limit amount **must** be entered in the
+  blank field before :guilabel:`Attendees`.
+- :guilabel:`Badge Dimension`: select a desired dimension for event badges. The options are:
+  :guilabel:`A4 foldable`, :guilabel:`A6`, or :guilabel:`4 per sheet`.
+- :guilabel:`Badge Background`: if desired, click the :guilabel:`Upload your file` button to upload
+  a custom background for event badges.
+
+When the fields in the event form have been adequately filled in, move on to the three tabs at the
+bottom of the event form for further customization.
+
+Those tabs are: :ref:`Tickets <create_events/event_form/tickets>`, :ref:`Communication
+<create_events/event_form/communication>`, :ref:`Questions <create_events/event_form/questions>`,
+and :ref:`Notes <create_events/event_form/notes>`.
+
+.. _create_events/event_form/tickets:
+
+Tickets tab
+-----------
+
+Create custom tickets (and ticket tiers) for events in the :guilabel:`Tickets` tab of an event form.
+
+.. image:: create_events/tickets-tab.png
+   :align: center
+   :alt: A typical tickets tab on an event form in the Odoo Events application.
+
+To create a ticket, click :guilabel:`Add a line` in the :guilabel:`Tickets` tab. Then, enter a name
```

Comment:
```suggestion
To create a ticket, click :guilabel:`Add a line` in the :guilabel:`Tickets` tab. Then, enter a name
```

Are we really just creating a new product here that will have the event ticket product type?

Reply:
Yes, that is my understanding -- since event registration products (tickets) **must** have the 'Event Ticket' product type in order to even appear in the 'Tickets' tab of an event form.